crates/matrix-sdk-crypto/src/olm/group_sessions/outbound.rs

```diff
@@ -672,8 +672,8 @@ mod tests {
 
         assert!(!session.expired());
         // FIXME: this might break on macosx and windows
-        let time = SystemTime::now() - Duration::from_secs(60 * 60);
-        session.creation_time = SecondsSinceUnixEpoch::from_system_time(time).unwrap();
+        session.creation_time =
+            modified_seconds_since_unix_epoch(|e| e - Duration::from_secs(60 * 60));
```

Comment:
This seems like a weird solution. Since we now have js `Date.now()` support in Ruma, this should now work instead:

```suggestion
        let now = SecondsSinceUnixEpoch::now();
        session.creation_time = SecondsSinceUnixEpoch(now.0 + 60 * 60);
```

Reply:
Neat! I wasn't aware that is in ruma now.

the exact could would be:
```rust
        let now = SecondsSinceUnixEpoch::now();
        session.creation_time = SecondsSinceUnixEpoch(now.0 - UInt::try_from(60u32 * 60u32).unwrap());
```

Now, we have a few other occasion where we currently use `seconds_since_unix_epoch`, do you mind it I clean all that up (including this instance) in a follow up PR? It just feels more atomically scoped to not do this here. I wanted to get rid of that annoying `instant` in `common` since I had to incept it.

Reply:
Yeah okay, can be a separate PR.